server/controllers/profileController.js

```diff
@@ -1,108 +1,151 @@
 const Profile = require("../models/ProfileModel");
-const User = require("../models/User");
 const mongoose = require("mongoose");
+const { capitalize, formatAddress } = require("../utils/helperFunctions");
 
-const toObjectId = mongoose.Types.ObjectId;
-
-// create new profile using given arguments
-
+// @route POST /profile
+// @access private
+// @desc Create a default profile of a logged-in user. Note: this profile is not a 'Sitter' profile
 exports.createProfile = async (req, res, next) => {
   try {
-    const { firstName, lasttName, phone, address, gender, description } =
+    const { firstName, lastName, gender, birth, phone, address, description } =
       req.body;
 
     const userId = req.user.id;
-    const user = await User.findById(userId);
-    if (!user) {
-      res.status(401).json({ message: "Not Authorized" });
-    }
 
-    if (
-      firstName === undefined ||
-      lasttName === undefined ||
-      phone === undefined ||
-      address === undefined ||
-      gender === undefined ||
-      description === undefined
-    ) {
-      res.status(401).json({ message: "Complete the form please" });
+    let profile = await Profile.findOne({ userId });
+
+    if (profile) {
+      res.status(404);
```

Comment:
`404` is when a resource is not found, wouldn't it be better to use a `409` here? :)

Reply:
Applied